File: javascript/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var nome = lines.shift();
var salario = parseFloat(lines.shift());
var vendas = parseFloat(lines.shift());
var comissao = vendas * 0.15;
var salarioFinal = salario + comissao;

console.log("TOTAL = R$ " + salarioFinal.toFixed(2));

File: C++/1009.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    string nome;
    double base, vendas;
    
    cin >> nome >> base >> vendas;
    
    cout << "TOTAL = R$ " << fixed << setprecision(2) << vendas * 0.15 + base << endl;
    return 0;
}